### PR TITLE
Update nodejs build doc

### PR DESCRIPTION
**Pull request by daniv-msft**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: doc/runtimes/nodejs.md [hunk 1]
```diff
@@ -46,8 +46,10 @@ The following process is applied for each build.
 2. Run custom build command if specified by `CUSTOM_BUILD_COMMAND`. This will skip running any package install commands 
    (`npm install` or `yarn install`) that Oryx originally included.
 3. Run custom run build command if specified by `RUN_BUILD_COMMAND`. This will not skip package install commands.
-4. Run `npm install` without any flags, which includes npm `preinstall` and
-   `postinstall` scripts and also installs devDependencies.
+4. Run `npm install`, which includes npm `preinstall` and `postinstall` scripts and also installs devDependencies.
+   The `--unsafe-perm` flag is added to the NPM install command, as well as the `--production` flag if some production
```
Comment (pauld-msft) [inline doc/runtimes/nodejs.md:50]: I actually removed the unsafe-perm flag in this PR, as it isn't needed now that the permissions are updated: https://github.com/microsoft/Oryx/pull/1759#pullrequestreview-1225116576

**Review by william-msft — DISMISSED** [review]

File: doc/runtimes/nodejs.md (hunk 1 above)
Comment (daniv-msft) [reply doc/runtimes/nodejs.md:50]: Ah good call, thanks. I'll update this.

Comment (daniv-msft) [reply doc/runtimes/nodejs.md]: Can't believe I forgot this one. Here is the update.

**Review by william-msft — APPROVED** [review]

**Review by cormacpayne — APPROVED** [review]
:shipit:

**Pull request merged**